### DISCUSSION TITLE
HSRB - Material Safety Data Sheet for the Lithium Battery

### DISCUSSION THREAD

**Discussion by adamgolding1** [HumanSupportRobot/QandA#54] [Q&A] [answered]
We are required to supply a Material Safety Data Sheet for the lithium battery to the airline for shipping the robot by air freight. Does anyone have a document with the battery specifications and materials?

The document would contain information such as:

Battery Model:
Manufacturuer:
Composition Information:
Hazard Identification:
Transport Information:

**Comment by hsr-admin** [comment, marked as answer]
Are the following documents sufficient?

[UN Report(E1826347) for 7UR18650E-5.pdf](https://github.com/HumanSupportRobot/QandA/files/11740158/UN.Report.E1826347.for.7UR18650E-5.pdf)
[Battery_SDS-IBT-00026-25-E.pdf](https://github.com/HumanSupportRobot/QandA/files/11740161/Battery_SDS-IBT-00026-25-E.pdf)

**Reply by adamgolding1** [reply]
That is perfect. Exactly what we needed. 

Thank you so much.
